index.html

```diff
@@ -1,28 +1,96 @@
 <!DOCTYPE html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Document</title>
+    <title>Luis Navarro's Portfolio</title>
   </head>
+
   <body>
-    <h1>Luis Navarro</h1>
     
-    <!-- The following text will not be semantic as to not get ahead of lesson 4.1, where that will be covered. -->
-    About
-    Luis Navarro is a web designer and WordPress developer who helps businesses and individuals with their digital marketing by creating functional and aesthetically pleasing websites.
+    <img src="images/profile-picture.jpg" alt="Luis Navarro">
+    <h1>Hello there! I am Luis. I help businesses build amazing websites</h1>
+    
+    <section>
+      <h1>About</h1>
+        <p>
+          My name is Luis Navarro. I am a web designer and developer with an engineering 
+          and cybersecurity background. I strive to simplify difficult tasks and make 
+          complex systems easy to use for everyone. Currently, I focus on building websites 
+          with WordPress.
+        </p>
+    </section>
+
+    <section>
+      <h1>Experience</h1>
+
+        <h2>Emeralt Web Studio</h2>
+          <h3>Owner, Web designer and developer</h3>
+            <ul>
```

Comment:
This all looks good. One point on the formatting: Usually these would all have the same indentation, like so:
```suggestion
      <h1>Experience</h1>

      <h2>Emeralt Web Studio</h2>
      <h3>Owner, Web designer and developer</h3>
      <ul>
```

If there were nested divs, then the indentation would be closer to what you have:
```suggestion
      <h1>Experience</h1>
      <div>
          <h2>Emeralt Web Studio</h2>
          <div>
              <h3>Owner, Web designer and developer</h3>
              <ul>
```